lib/stat_tracker.rb

```diff
@@ -22,4 +22,29 @@ def self.read_from_file(file)
     file_open.read
   end
 
+# ************* LeagueStatistics *************
+
+  def match_data_with_header(file)
+    self.instance_variable_get(file).transpose
+  end
```

Comment:
If I'm understanding this one correctly, it changes our data so that instead of being an array of rows, it becomes an array of columns. I'm unsure of why we'd need self.instance_variable_get, could it just be
```ruby
def match_data_with_header(data)
end